### DISCUSSION TITLE
Should we bother with keeping the Zachtek (type-3) message option?

### DISCUSSION THREAD

**Discussion by EngineerGuy314** [EngineerGuy314/pico-WSPRer#31] [General]
Does anyone plan on using the Zachtek type protocol instead of or in addition to the U4B protocol? Wondering if I should drop it completely. When using both  TELEN #1 and #2 telemetry it won't be possible anyway. Although I suppose its possible to use just telen#1, and still do Zachtek.

Right now the current version has telen #1 and #2 forced on, for testing. I need to expand the command-line user setup menu to be able to enable/disable telen, and do preselected functions for them (high resolution ADC inputs, I2C registers..). With all that added complexity might be time to do a revamp of the whole setup system. Maybe something like the U4B has where you can actually navigate with TAB and arrow keys? My other thought was having all the config in a text file that you upload to the pico when you upload the .uf2 file. not sure if thats even possible.

**Comment by whallmann** [comment]
No, i don't use Zachtek. I prefere the U4B.  cu Wolf

EngineerGuy314 ***@***.***> schrieb am Mo., 22. Juli 2024,
16:37:
